DirectProgramming/DPC++/SparseLinearAlgebra/merge-spmv/README.md

```diff
@@ -11,11 +11,11 @@ For comprehensive instructions regarding DPC++ Programming, go to https://softwa
 | What you will learn               | The Sparse Matrix Vector sample demonstrates the following using the Intel&reg; oneAPI DPC++/C++ Compiler <ul><li>Offloading compute intensive parts of the application using lambda kernel</li><li>Measuring kernel execution time</li></ul>
 | Time to complete                  | 15 minutes
 
+
 ## Purpose
-Sparse linear algebra algorithms are common in HPC, in fields as machine learning and computational science. In this sample, a merge based sparse matrix and vector multiplication algorithm is implemented. The input matrix is in compressed sparse row format. Use a parallel merge model enables the application to efficiently offload compute intensive operation to the GPU.
+Sparse linear algebra algorithms are common in HPC, in fields as machine learning and computational science. In this sample, a merge based sparse matrix and vector multiplication algorithm is implemented. The input matrix is in compressed sparse row format. Use a parallel merge model enables the application to efficiently offload compute intensive operation to the GPU. For comparison, the application is run sequentially and parallel with run times for each displayed in the application's output. The device where the code us run is also identified.
```

Comment:
"is run sequentially and parallel" -> " is run sequentially and parallelly"
"the code us run" -> "the code is run"